packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -556,6 +555,9 @@ apiDescribe('Database transactions', (persistence: boolean) => {
         })
         .then(() => {
           // Now all transaction should be completed, so check the result.
+          // There should be a maximum of 3 retries: once for the 2nd update,
+          // and twice for the 3rd update.
+          expect(started).to.be.lessThan(7);
```

Comment:
Why is this `7` instead of `6`?  Also, on Android:
1) You renamed started to counter which was a good change.
2) You added a "There should be 3 initial transaction runs." comment above which was a good change.

Reply:
On android it's <= 6, but I don't think `expect` has a `lessThanOrEqualTo` equivalent.

Reply:
Oh!  Right, that makes sense. :)